lib/utils.pm

```diff
@@ -430,8 +430,10 @@ sub workaround_type_encrypted_passphrase {
     # nothing to do if the boot partition is not encrypted in FULL_LVM_ENCRYPT
     return if get_var('UNENCRYPTED_BOOT');
     return if !get_var('ENCRYPT') && !get_var('FULL_LVM_ENCRYPT');
+    # for Leap 42.3 and SLE 12 codestream the boot partition is not encrypted
+    # Only aarch64 needs separate handling
```

Comment:
I did not understand yet why aarch64 should be different. It's not the architecture that actually differs I assume but really something like UEFI or maybe something else on top.

Reply:
I guess it's related to lvm + encryption support on aarch64 rather than just pure UEFI.

Reply:
@okurz rather it's ppc64 that needs the special handling.